### DISCUSSION TITLE
使用websocket的时候报错

### DISCUSSION THREAD

**Discussion by babizhu** [alex8088/electron-vite#274] [General]
Refused to connect to 'http://localhost:9099/socket.io/?loginUserNum=79&EIO=4&transport=polling&t=OfZUbtx' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'connect-src' was not explicitly set, so 'default-src' is used as a fallback.

**Comment by alex8088** [comment]
csp问题，可在index. html中设置

**Reply by babizhu** [reply]
搞定，感谢
